hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/webapp/TestRouterWebServices.java

```diff
@@ -200,6 +200,12 @@ public void testRouterWebServicesE2E() throws Exception {
 
     ContainerInfo containerInfo = getContainer(user);
     Assert.assertNotNull(containerInfo);
+
+    Response response19 = updateSchedulerConfiguration(user);
+    Assert.assertNotNull(response19);
```

Comment:
There's nothing more we can check?

Reply:
Thank you very much for your help reviewing the code, your suggestion is right, the current unit tests are a little less, I personally think that the purpose of this pr is to introduce new methods, and the specific implementation will be done in a separate pr, for this pr, I think it is enough at present .